MVP Module/Source/Feed Presentation/FeedPresenter.swift

```diff
@@ -9,21 +9,28 @@ protocol FeedLoadingView {
 	func display(_ viewModel: FeedLoadingViewModel)
 }
 
+protocol FeedErrorView {
+	func display(_ viewModel: FeedErrorViewModel)
+}
+
 protocol FeedView {
 	func display(_ viewModel: FeedViewModel)
 }
 
+typealias FeedViewFlow = FeedView & FeedErrorView
+
 final class FeedPresenter {
-	private let feedView: FeedView
+	private let feedView: FeedViewFlow
 	private let loadingView: FeedLoadingView
 
-	init(feedView: FeedView, loadingView: FeedLoadingView) {
+	init(feedView: FeedViewFlow, loadingView: FeedLoadingView) {
 		self.feedView = feedView
 		self.loadingView = loadingView
 	}
```

Comment:
Following from my previous comment, we can inject a different object to handle displaying errors:
```suggestion
	init(feedView: FeedView, loadingView: FeedLoadingView, errorView: FeedErrorView) {
		self.feedView = feedView
		self.loadingView = loadingView
		self.errorView = errorView
	}
```

Reply:
done